src/rust/engine/process_execution/src/docker_tests.rs

```diff
@@ -73,9 +71,8 @@ macro_rules! setup_docker {
 #[tokio::test]
 #[cfg(unix)]
 async fn stdout() {
-  let docker = setup_docker!();
+  let _docker = setup_docker!();
```

Comment:
`setup_docker!()` is now redundant, right? It looks like its only use is for macOS, that we eagerly skip the test if Docker is not running. If so, I can rename this to be `maybe_skip_on_macos` or something more clear

Reply:
To be done in a follow-up. I want to figure out first what CI variable to check for so we can avoid the issue you saw with it being skipped when doing local development on macOS.

Reply:
Sg. And I realize this also is helpful to give more clear error messages when Docker is not set up on Linux